src/frontend/screens/Quests/index.module.scss

```diff
@@ -34,20 +34,34 @@
 }
 
 .detailsWrapper {
-  width: 800px;
+  width: 650px;
   max-width: unset !important;
 
   background-color: var(--color-neutral-800);
   border: none;
 
-  @media (max-width: 1300px) {
-    width: 540px;
+  //Steam Deck
+  @media (max-width: 1280px) {
+    width: 600px;
+  }
+
+  @media (max-width: 1024px) {
+    width: 390px;
   }
```

Comment:
You could use the preset media queries that we have in the ui lib `src\styles\utilities\_variables.scss` here. That way the breakpoints are more readable and defined in one place if we want to change them across the app. If you ctrl+f for some of them, you should be able to find some examples in the client or the ui lib of their usage.

Reply:
I checked these, but any of them match the sizes I tested for this quest screen. To be able to show the search bar and have a minimal size to show the content in each component piece for 1280 and 1024 screens I had to create these. We can add this to the variables in the ui lib and reuse them here or keep like this as I believe we will not use this specificity in other place, wdyt? @BrettCleary